contracts/SimpleAssetsExchange.sol

```diff
@@ -65,6 +85,10 @@ contract SimpleAssetsExchange is Ownable, IAssetsExchange {
     uint256 amountOut = _amount * priceProvider.getPrice(_asset) / 1 ether;
```

Comment:
This won't work as the calculated amountOut may be different than the amount received from Pangolin